tests/unit_tests/pyanaconda_tests/core/test_startup_utils.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2021  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Test code from the startup_utils.py. Most of the parts are not
+# easy to test but we can try to improve that.
+#
+
+import unittest
+
+from unittest.mock import patch, mock_open, Mock
+
+from anaconda import print_dracut_errors
+
+
+class StartupUtilsTestCase(unittest.TestCase):
+
+    def test_print_dracut_errors(self):
+        logger_mock = Mock()
+
+        with patch("anaconda.open", mock_open(read_data="test text")) as m:
+            print_dracut_errors(logger_mock)
+
+            m.assert_called_once_with("/run/anaconda/initrd_errors.txt", "rt")
+
+            logger_mock.warning.assert_called_once_with(
+                """
+############## Installer errors encountered during boot ##############
+test text
+############ Installer errors encountered during boot end ############""")
```

Comment:
You can indent the lines of the string and call `dedent` to fix it. The tests are more readable that way.